tasks/custom.yml

```diff
@@ -0,0 +1,68 @@
+---
+
+- name: Download custom modules source tar
+  become: yes
+  get_url:
+    url: "{{ item.source }}"
+    dest: "/opt/{{ item.name }}.tar.gz"
+  loop: "{{ nginx_custom_modules }}"
+
+- name: Extract custom modules
+  become: yes
+  unarchive:
+    src: "/opt/{{ item.name }}.tar.gz"
+    dest: /opt
+    remote_src: yes
+  loop: "{{ nginx_custom_modules }}"
+
+- name: Collect current nginx binary version
+  shell: "nginx -v 2>&1 | grep -o '[0-9.]*$'"
+  register: nginx_version
+  failed_when: not nginx_version.stdout
+  changed_when: false
+
+- name: Ensure tmp location exists for download
+  file:
+    path: /tmp/nginx
```

Comment:
I think it would make sense for this to be moved to the `/opt` directory like the custom modules.

Reply:
Also, this is probably worth turning into a variable given how much it's used throughout the file.